### PR TITLE
completed copy function

### PR DESCRIPTION
**Pull request by musojon1**

### CODE REVIEW

File: src/codeflask.css [hunk 1]
```diff
@@ -28,6 +28,24 @@
 	border-top-left-radius: 5px;
 }
 
+.editorjs-codeFlask_Wrapper .editorjs-codeFlask_CopyButton{
+	position: absolute;
+	z-index: 2;
+	line-height: 20px;
+	background-color: #dcdfe6;
+	padding: 5px;
```
Comment (samandar-boymurodov) [inline src/codeflask.css:34]: padding: 5px 10px

File: src/codeflask.js [hunk 2]
```diff
@@ -138,12 +138,25 @@
...
     langdisplay.classList.add('editorjs-codeFlask_LangDisplay')
+    let coppyButton = document.createElement('button')
+    coppyButton.classList.add("editorjs-codeFlask_CopyButton");
+    coppyButton.innerHTML = this._copySvg;
 
     langdisplay.innerHTML = this.data.language
-
+    coppyButton.addEventListener("click",async () => {
+      coppyButton.innerHTML =
+        '<svg fill="#000000" height="1em" width="1em" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 512.42 512.42" xml:space="preserve"><g><g><path d="M509.313,67.45c-4.16-4.16-10.88-4.267-15.04-0.107L134.06,422.33l-115.2-121.6c-3.733-4.48-10.453-5.12-15.04-1.387 c-4.48,3.733-5.12,10.453-1.387,15.04c0.32,0.32,0.533,0.64,0.853,0.96l122.667,129.493c1.92,2.133,4.693,3.307,7.573,3.307h0.213 c2.773,0,5.44-1.067,7.467-3.093l368-362.667C513.473,78.33,513.473,71.61,509.313,67.45z"/></g></g></svg>';
```
Comment (samandar-boymurodov) [inline src/codeflask.js]: Can we keep svg strings in vars.js

**Review by samandar-boymurodov — CHANGES_REQUESTED** [review]

**Comment by samandar-boymurodov** [comment]
Let's increment the patch version by one. So that we can publish with a new version.

File: src/codeflask.js (hunk 2 above)
Comment (musojon1) [reply src/codeflask.js]: Yes,we can. I'll do it

**Review by samandar-boymurodov — APPROVED** [review]

**Pull request merged**